client/components/relatedWidget/Outfit.jsx

```diff
@@ -44,30 +36,20 @@ class Outfit extends React.Component {
     });
   }
 
-  onSelect(key) {
-    this.setState({ selected: key });
-  }
-
```

Comment:
Function used to highlight product cards that isn't necessarily needed right now